Mixmatch/.ipynb_checkpoints/README-checkpoint.md

```diff
@@ -0,0 +1,143 @@
+# MixMatch - A Holistic Approach to Semi-Supervised Learning
```

Comment:
Is the "checkpoint" text in filename expected?

Reply:
I think I should not push the .ipynb_checkpoints folder to github. This folder is used for checkpoints from jupyter lab.